chapters/en/module3.md

```diff
@@ -1,111 +1,226 @@
+**Git** has a distinct step of **adding** files to the staging area because:
 
 <choice>
-<opt text='file.txt' >
+<opt text='Not all changes we make (i.e., files we create or edit) are ones that we want to push to our remote GitHub repository.'>
 </opt>
-<opt text='otherfile.txt'>
+<opt text='It allows us to edit multiple files at once, but associate particular commit messages with particular files (so that the commit messages can more specifically reflect the changes that were made).'>
 </opt>
-<opt text='mynewfile.txt' correct='true'>
+<opt text='This is technically required of all version control software.'>
+</opt>
+<opt text='A and C'>
+</opt>
+<opt text='A and B' correct='true'>
 </opt>
-
 </choice>
+
 </exercise>
 
-<exercise id='7' title='git commit' type='slides, video'>
-<slides source='module1/module1_00' shot='0' start='3:42' end='4:35'> </slides>
+<exercise id='7' title='Committing changes to a local repository' type='slides, video'>
+<slides source='module3/module3_04' shot='0' start='3:42' end='4:35'> </slides>
 </exercise>
 
-<exercise id='8' title='commit message'>
+<exercise id='8' title='A commit in a bottle'>
 
-What of the following commit messages is more informative?
+**Question 1**
 
-a. x
-b. x
-c. x
+**True or false**
 
-<choice>
-<opt text='a and b' >
+> When we **commit** our changes to Git, the snapshot of changes, the commit message, the time and date stamp and the user who committed the changes are all saved to the Git history on GitHub.
+
+<choice id='1'>
+<opt text='true'>
+
+Solution is incorrect. Committing your changes only puts them in the Git history on the local computer you are working on (i.e., your workspace on the JupyterHub or your laptop). To get the changes on GitHub you need to do an additional step of pushing the changes to the remote repository on GitHub.
+
+</opt>
+<opt text='false' correct='true'>
+
+"Success! You're right! The changes (and all the associated information) are not yet on GitHub, they are only in the Git history on the local computer you are working on (i.e., your workspace on the JupyterHub or your laptop)."
+
+</opt>
+</choice>
+
+**Question 2**
+
+When working on a project that is under version control you should `commit` to you local Git repository:
+
+<choice id='2'>
+<opt text='After every new unit/addition/fix you add to the project' correct='true'>
+</opt>
+<opt text='2-3 times'>
+</opt>
+<opt text='only once, shortly before the deadline'>
+</opt>
+<opt text='obsessively, like every 5 minutes'>
+</opt>
+</choice>
+
+**Question 3**
+
+Which command(s) below would save the changes of `README.md` to my local Git repository? 
+
+<choice id='3'>
+<opt text='`git init README.md`  `git commit -m "Add authors and copyright"`' >
 </opt>
-<opt text='b and c'>
+<opt text='`git commit -m "Add authors and copyright"`'>
 </opt>
-<opt text='c and d' correct='true'>
+<opt text='`git add README.md` `git commit -m "Add authors and copyright"`'  correct='true'>
```

Comment:
```suggestion
<opt text='`git add README.md` and `git commit -m "Add authors and copyright"`'  correct='true'>
```

These don't show as rendered code, maybe we need the `<code>` tag instead of backtick?